uisdk/src/main/java/com/karhoo/uisdk/util/extension/QuoteVehicleExt.kt

```diff
@@ -15,3 +15,13 @@ fun QuoteVehicle.categoryToLocalisedString(context: Context): String? {
         else -> vehicleClass
     }
 }
+
+fun QuoteVehicle.typeToLocalisedString(context: Context): String? {
```

Comment:
Using this extension, you can easily implement it in other paces which we are using Vehicle Class.